modules/rules_kustomize/0.3.1/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "integrity": "sha256-165Nnv7l0PxKZ6iuhj8UlyluvK6RLKQ7+UXE1wQ3yEQ=",
+    "strip_prefix": "rules_kustomize-0.3.1",
+    "url": "https://github.com/bisontrails/rules_kustomize/archive/refs/tags/v0.3.1.tar.gz",
```

Comment:
See https://github.com/orgs/community/discussions/46034, this kind of URL is no longer guaranteed to be stable.

Reply:
Yes, publishing and referring to the source archive is on my list of pending tasks.